src/main/frontend/client/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Node Group Project</title>
+    <link rel="stylesheet" href="./public/style.css">
+  </head>
+
+  <body>
+    <div id="app"></div>
+
+    <script src="//code.jquery.com/jquery-2.1.3.min.js"></script>
+    <script src="/bundle.js"></script>
+  </body>
+</html>
```

Comment:
We should not need this file, as the `src/main/resources/static/index.html` is doing this for us. Please remove!